diff --git a/dist/exploration.module.js b/dist/exploration.module.js
index 714ce87eaa41ce4845da0b4f935fb308b7a20725..e207435d26492ac1b34f57823eba4cae54c2d6cc 100644
--- a/dist/exploration.module.js
+++ b/dist/exploration.module.js
@@ -2,7 +2,7 @@
 Object.defineProperty(exports, "__esModule", { value: true });
 exports.ExplorationModule = void 0;
 const core_1 = require("@nestjs/core");
-const internal_core_module_1 = require("@nestjs/core/injector/internal-core-module");
+const internal_core_module_1 = require("@nestjs/core/injector/internal-core-module/internal-core-module.js");
 const spelunker_messages_1 = require("./spelunker.messages");
 class ExplorationModule {
     static explore(app, opts) {
